portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
+    messages.add("hello");
+    messages.add("how are you");
+    messages.add("good morning");
+  }
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello Jenny!</h1>");
```

Comment:
I'm not sure if this works? You're setting the ContentType to "text/html;" but returning both "html" and "json" types in `doGet`. I think it would be safer to just return GSON here (which I think was the exercise)

`@Override`
`public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {`
`   Gson gson = new Gson();`
`   String json = new Gson().toJson(messages);`
`   response.setContentType("application/json;");`
`   response.getWriter().println(json);`
`}`